pkg/openstack/loadbalancer.go

```diff
@@ -618,6 +619,39 @@ func (lbaas *LbaasV2) updateFloatingIP(ctx context.Context, floatingip *floating
 	return floatingip, nil
 }
 
+func (lbaas *LbaasV2) updateFloatingIPTag(ctx context.Context, floatingip *floatingips.FloatingIP, Tag string, delete bool) error {
```

Comment:
`delete` is a reserved golang keyword, I'd rather use `del`, or even better, use `tag` as a pointer. if the pointer is `nil`, then delete

```suggestion
func (lbaas *LbaasV2) updateFloatingIPTag(ctx context.Context, fip *floatingips.FloatingIP, tag *string) error {
```

Reply:
Need to search and delete based on tags, and  cannot delete all the tags , pointer cannot be used,